### DISCUSSION TITLE
Time domain problem of training

### DISCUSSION THREAD

**Discussion by kana-gpu** [lululxvi/deepxde#1841] [General]
Hi all.
I need to use neural networks to solve a partial differential equation. However, during the process of network training, problems arose.
In the time domain I defined, the training results were good in the first half of the time, but problems arose later on. In order to conduct subsequent training, I segmented the later period more finely, and after using the numerical solution of the left endpoint of each segmented time domain as the observation constraint, I was able to obtain better values.
However, for example, if I use the numerical solution at time point 0.5 as the observation condition, I can obtain a better result of 0.6, but the errors in the results of 0.7 and 0.8 are relatively large.
My ultimate goal is to be able to directly obtain the results I want in the time domain without using observation points, or in other words, there is a way to expand its computational domain while using observation points (for example, by training the observation values at 0.5s, we can obtain values at 0.9s and 1.0s). Or is there any code available for segment by segment training in Deepxde for learning.
Thank you in advance for your help.

**Comment by lululxvi** [comment]
You can split the time domain and train one by one. For later time domains, you can use the PINN solution from the previous domain as the data. There are many papers using this, and you can find more details.
